CHANGES.md

```diff
@@ -30,6 +30,8 @@
 
 - Fix a bug where a package which had a single version that built with dune and got selected by the solver
   would be reported has having no version building with dune. (#245, @Leonidas-from-XIV)
+- Fix the solver so it does not select beta versions of the compiler unless it
+  has to. (#269, @NathanReb)
```

Comment:
Could you reword this a little? The current wording is a bit unclear, the reader is probably wondering "when does it have to then?". Specifying it like "it does not select beta version of the compiler unless forced by version constraints or on the command line" could be clearer.

Reply:
Yeah sure, it is not very clear indeed!

Reply:
I think it's better know so I'll merge don't hesitate if you think it's still not clear enough, I'll be happy to update it separately!

We should do a brief review of the changelog before the release by the way to make sure everything is clear.

Reply:
A review sounds good. Sometimes the entries make sense when writing the PR but afterwards it might be less so.